pkg/console/dashboard_panels.go

```diff
@@ -249,6 +257,65 @@ func getVIP() string {
 	return outStr
 }
 
+func syncNodeIP(ctx context.Context, g *gocui.Gui) {
+	// sync IP at the beginning
+	doSyncNodeIP(g)
+
+	syncDuration := 30 * time.Second
```

Comment:
Does it mean that  If I change the node IP after beginning, I need  30 seconds to check the current IP?
If so, is the value too long?

Reply:
Currently, we disallow users to change node IP.
These 30 seconds should be only needed for booting up if the network is preparing.

Reply:
OK, keep in mind that. Thanks for explaining :)